grid-proxy/pkg/types/types.go

```diff
@@ -29,6 +29,7 @@ type Counters struct {
 	Twins             int64            `json:"twins"`
 	Contracts         int64            `json:"contracts"`
 	NodesDistribution map[string]int64 `json:"nodesDistribution" gorm:"-:all"`
+	GPUs              int64            `json:"gpus"`
```

Comment:
should we unify the json to `num_gpus` here and in 191?

Reply:
I don't think so, since everything in that response is technically a num as well